backend/.dockerignore

```diff
@@ -0,0 +1,3 @@
+.dockerignore
```

Comment:
Is this line needed? I always assumed `.dockerignore` would be automatically ignored, but correct me if I'm wrong.